src/babel.js

```diff
@@ -28,6 +26,11 @@ const options = config.files.size
 		],
 		babelrc: false,
 		configFile: false,
+		overrides: config.files.size
```

Comment:
What about using `overrides`, set to the project config, if any?

Currently working with a sparse project config file:

```javascript
// babel.config.js in project root.

module.exports = function(api) {
  api.cache(true);

  return {
    env: {
      test: {
        plugins: [
          [
            '@babel/plugin-transform-react-jsx',
            {
              runtime: 'automatic',
            },
          ],
        ],
      },
    },
  };
};
```

Reply:
Do remember you need to also set the `importSource` if you're using the automatic transform.

Reply:
> Do remember you need to also set the `importSource` if you're using the automatic transform.

Thx. I saw your earlier comment about that. 

I can't tell you why, but in my project tests are now running for me with automatic runtime, even without `importSource`. Adding it to `babel.config.js` doesn't make any difference that I can see.

Reply:
It works as we alias `react/jsx-runtime` to `preact/jsx-runtime`.

However, I'd caution against relying on that as module resolution can be somewhat finicky and the main concern is that you end up consuming `preact/compat/jsx-runtime` (as is a common-ish issue, though not one that's present here at the moment, to be clear). That module is `preact/compat` + `preact/jsx-runtime`, so while it'll usually work, `preact/compat` has side effects that could affect your tests.

I'd recommend being explicit and not relying upon aliases that are internal to the preset. It'll also make your Babel config a lot more portable; you can take that and copy/paste it into other projects without worry if you're not relying on externals like aliases to sort paths out.

Just my 2¢, feel free to ignore.